crates/kornia-io/src/functional.rs

```diff
@@ -29,6 +41,10 @@ use std::path::Path;
 /// assert_eq!(image.rows(), 195);
 /// assert_eq!(image.num_channels(), 3);
 /// ```
+#[deprecated(
+    since = "0.1.12",
+    note = "Use explicit typed readers (jpeg::read_image_jpeg_rgb8, png::read_image_png_rgb8, etc.) instead. For Python, use read_image() which auto-detects format and type."
```

Comment:
I think we should keep the rust side purely rust side, and not tell python users that they can use read_image()
```suggestion
    note = "Use explicit typed readers (jpeg::read_image_jpeg_rgb8, png::read_image_png_rgb8, etc.) instead."
```